packages/ckeditor5-font/src/ui/colorui.ts

```diff
@@ -139,10 +139,18 @@ export default class ColorUI extends Plugin {
 				editor.editing.view.focus();
 			} );
 
+			dropdownView.on( 'colorChange', ( evt, data ) => {
```

Comment:
`colorChange` event on color dropdown is very generic. Note that it should also be fired when selecting regular colors, while it doesn't.

Given that `_colorPickerView` member will become a public property it's better to just listen here on `dropdownView.colorPickerView` directly (with a check if it exists, because it doesn't have to).